### PR TITLE
feat: updated verifiable controllers with the new methods

### PR DESCRIPTION
**Pull request by ToJen**
Signed-off-by: Tomisin Jenrola <tomisin.jenrola@securekey.com>

**Title:**
Update verifiable controllers

**Description:**
Closes https://github.com/hyperledger/aries-framework-go/issues/2034

Summary:
- Implement the two methods recently added to the verifiable store.
- updated the command controller
- updated the rest controller

### CODE REVIEW

**Comment by fqutishat** [comment]
@ToJen where is the integration test

**Comment by ToJen** [comment]
@fqutishat 
Should I create a new feature file here: https://github.com/hyperledger/aries-framework-go/tree/master/test/bdd/features?

It appears verifiable has no existing bdd tests. Or should I add a new scenario in `issue_credential.feature`

**Comment by fqutishat** [comment]
> @fqutishat
> Should I create a new feature file here: https://github.com/hyperledger/aries-framework-go/tree/master/test/bdd/features?
> 
> It appears verifiable has no existing bdd tests. Or should I add a new scenario in `issue_credential.feature`

@ToJen https://github.com/hyperledger/aries-framework-go/tree/master/test/aries-js-worker

File: pkg/controller/rest/verifiable/operation.go [hunk 1]
```diff
@@ -240,3 +244,33 @@ func (o *Operation) GeneratePresentation(rw http.ResponseWriter, req *http.Reque
 func (o *Operation) GeneratePresentationByID(rw http.ResponseWriter, req *http.Request) {
 	rest.Execute(o.command.GeneratePresentationByID, rw, req.Body)
 }
+
+// RemoveCredentialByName swagger:route POST /verifiable/credential/remove/name/{name}
```
Comment (soluchok) [inline pkg/controller/rest/verifiable/operation.go:248]: this comment should be like this:
`RemoveCredentialByName swagger:route POST /verifiable/credential/remove/name/{name} verifiable RemoveCredentialByNameReg `
where `verifiable` groups your APIs by name `verifiable`
and `RemoveCredentialByNameReq` describes the request 

the swagger spec will be incorrect, you need to check it by `make run-openapi-demo` https://github.com/hyperledger/aries-framework-go/blob/master/docs/rest/openapi_demo.md

NOTE: it is related to every operation (models are missing)

File: pkg/controller/rest/verifiable/operation.go [hunk 2]
```diff
@@ -240,3 +244,33 @@ func (o *Operation) GeneratePresentation(rw http.ResponseWriter, req *http.Reque
...
+//
+// Removes a verifiable credential by name.
+//
+// Responses:
+//    default: genericError
+//        200: emptyRes
```
Comment (soluchok) [inline pkg/controller/rest/verifiable/operation.go:254]: also, I see that `emptyRes` is not working properly.
the structure should be with the body inside.
e.g 
```
// emptyRes model
//
// swagger:response emptyRes
type emptyRes struct {
Body struct {}
}
```
I also noticed that we are using that structure which means we need to duplicate it.
One structure should be for swagger another structure should be for `Go` usage.

Comment (soluchok) [reply pkg/controller/rest/verifiable/operation.go:254]: the working solution can be like this:
```
type emptyRes struct {}

// emptyResponse model
//
// swagger:response emptyResponse
type emptyResponse struct { // nolint:unused,deadcode
	// in: body
	Body emptyRes
}
```
In that case, you need to replace `200: emptyRes` with `200: emptyResponse`

File: pkg/controller/rest/verifiable/operation.go (hunk 1 above)
Comment (ToJen) [reply pkg/controller/rest/verifiable/operation.go:248]: i'll add these

File: pkg/controller/rest/verifiable/operation.go (hunk 2 above)
Comment (ToJen) [reply pkg/controller/rest/verifiable/operation.go]: updated to reflect this

**Pull request merged**